docker-storage-setup.sh

```diff
@@ -823,6 +830,13 @@ setup_storage() {
   fi
 }
 
+reset_storage() {
+    if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
+	reset_lvm_thin_pool
+    fi
+    rm -f $DOCKER_STORAGE
```

Comment:
Yes we do.  `rpm -ql docker |grep docker-storage`.  Also on an Atomic Host system, I clearly see `/usr/etc/sysconfig/docker-storage`.